lightning-invoice/src/ser.rs

```diff
@@ -365,22 +365,22 @@ impl Base32Len for Fallback {
 	}
 }
 
-impl ToBase32 for Route {
+impl ToBase32 for RouteHint {
 	fn write_base32<W: WriteBase32>(&self, writer: &mut W) -> Result<(), <W as WriteBase32>::Err> {
 		let mut converter = BytesToBase32::new(writer);
 
 		for hop in self.iter() {
-			converter.append(&hop.pubkey.serialize()[..])?;
-			converter.append(&hop.short_channel_id[..])?;
+			converter.append(&hop.src_node_id.serialize()[..])?;
+			converter.append(&hop.short_channel_id.to_be_bytes())?;
```

Comment:
Oops, I think this (and the from_be_bytes calls) bumps lightning-invoice's MSRV - we should change the CI to also build+test lightning-invoice on 1.30 not just lightning.